### PR TITLE
Stop Jailers of Prudence, Hope, Justice, and Love from instantly despawning the second they go unclaimed.

### PR DESCRIPTION
**Pull request by TeoTwawki**
Note the comment in Jailer of Prudence's script : I don't have time to research this before I go offline for the weekend, but its no more broken than before I touched it and can easily be adjusted.

If anyone has any input before tomorrow afternoon I can do a quick amend for this if needed.

### CODE REVIEW

File: scripts/zones/AlTaieu/mobs/Jailer_of_Prudence.lua [hunk 1]
```diff
@@ -90,4 +89,12 @@ function onMobDeath(mob, player, isKiller)
...
     end;
-end;
\ No newline at end of file
+end;
+
+-----------------------------------
+-- onMobDedpawn
+-----------------------------------
+
+function onMobDedpawn(mob)
```
Comment (xdemolish) [inline scripts/zones/AlTaieu/mobs/Jailer_of_Prudence.lua]: dedpawn

Comment (TeoTwawki) [inline scripts/zones/AlTaieu/mobs/Jailer_of_Prudence.lua]: I imagine you delivering this news in your best deadpan snarker voice.

**Comment by Deadwing888** [comment]
Right, so here's how Jailer of Prudence works:

Prudences will spawn with 60 seconds of flee, but if the popper pops from outside the alliance and manages to flee all the way to the southern updraft to zone, the Prudence that was following them will begin pathing slowly back to the ??? and will depop after 3 minutes.

Once it depops, the other prudence will open its mouth and gain 100% triple attack. If it manages to wipe the claimed alliance in this state it will head to where the other prudence depopped, and will itself depop as soon as it gets there.

In every other circumstance they are sharing hate right up until the wipe at which point they will naturally depop together at the ???

**Comment by TeoTwawki** [comment]
@Deadwing888 thanks for clearing that up. I've moved the mouth open logic from death to despawn, since despawn will be called after death this covers both scenarios.

File: scripts/zones/AlTaieu/npcs/qm2.lua [hunk 2]
```diff
@@ -1,25 +1,26 @@
...
     -- Trade the Second Virtue, Deed of Moderation, and HQ Xzomit Organ
-    --[[if (GetMobAction(16912839) == 0 and trade:hasItemQty(1853,1) and trade:hasItemQty(1854,1) and trade:hasItemQty(1785,1) and
+    if (GetMobAction(16912839) == 0 and trade:hasItemQty(1853,1) and trade:hasItemQty(1854,1) and trade:hasItemQty(1785,1) and
     trade:getItemCount() == 3) then
         player:tradeComplete();
-        SpawnMob(16912839,900):updateClaim(player); -- Spawn Jailer of Justice
-    end]]
+        SpawnMob(16912839):updateClaim(player); -- Spawn Jailer of Justice
+    end
+    ]]
```
Comment (xdemolish) [inline scripts/zones/AlTaieu/npcs/qm2.lua]: why's this commented out? better to just remove if it's wrong/unneeded

Comment (TeoTwawki) [inline scripts/zones/AlTaieu/npcs/qm2.lua]: Yeah I just moved the placement of the comment block to be what I thought was more legible/less likely to have its indents f\*cked later

**Comment by TeoTwawki** [comment]
*nudge*

**Pull request merged**